highfidelity/hifi
# Use ScrollBars from QQC2, which supports mouse and touch events

Test plan:
1. Open Tablet
2. Open Running scripts
3. Make sure scroll bar works using mouse/stylus/lasers
4. Open any File dialog, make sure its possible scroll lists using scrollbars
5. Click on empty spaces on scroll bars, make sure lists gets scrolled
6. Repeat same for Desktop mode

Known issues:
leaving Tablet with pressed button and release outside Tablet will cause stick to mouse cursor. Its common problem not only for scroll bars but for whole Tablet area (just try to click and scroll and list moving mouse outside Tablet)
